pettingzoo/mpe/simple_spread/simple_spread.py

```diff
@@ -164,6 +164,8 @@ def reward(self, agent, world):
         rew = 0
         if agent.collide:
             for a in world.agents:
+                if(agent == a):
+                    continue
                 if self.is_collision(a, agent):
                     rew -= 1
```

Comment:
Could you change this to `rew -= 1.0 * (self.is_collision(a, agent) and a != agent)`? This just makes it cleaner.

Reply:
Yeah sure